src/api/test/functional/interconnect_test.rb

```diff
@@ -395,7 +395,7 @@ def test_invalid_operation_to_remote
     put '/source/RemoteInstance:BaseDistro2.0/pack/_meta', params: '<package name="pack" project="RemoteInstance:BaseDistro2.0">
            <title/><description/></package>'
     assert_response 403
-    assert_xml_tag tag: 'status', attributes: { code: 'create_package_no_permission' }
+    assert_xml_tag tag: 'status', attributes: { code: 'update_project_not_authorized' }
```

Comment:
Please squash this commit into where this got introduced:-)

And a description in the commit message why this changed would be nice. In any case I would document this in our release notes, since this changes our API.
Though I would double check if this change is correct (because it changes the API). It looks like this was / is an indirect way of creating a package and even if this was incorrect it now says "update_project_*" while the request url looks like it's send to the update package route.

Reply:
done